README.md

```diff
@@ -124,12 +124,39 @@ request.applePayDisabled = !canMakePayments
 
 ### 3D-Secure + Drop-In
 
-The new Drop-In supports 3D-Secure verification. If you have enabled 3D-Secure in the control panel, then just enable it in the BTDropInRequest and set an amount.
+The new Drop-In supports 3D-Secure verification. If you have enabled 3D-Secure in the control panel, then just enable it in the BTDropInRequest and set an amount. Then, create a BTThreeDSecureRequest() object and set as many fields on it as possible; the more fields that are set, the less likely it is for a user to be presented with a challenge. Make sure to attach this object to the BTDropInRequest before use.
```

Comment:
```suggestion
The new Drop-In supports 3D-Secure verification. If you have enabled 3D-Secure in the control panel, enable it in the BTDropInRequest, too, and set an amount. Then, create a BTThreeDSecureRequest() object, setting as many additionalInformation fields as possible; the more fields that are set, the less likely you will encounter a decline or the user will be presented with a challenge. Make sure to attach this object to the BTDropInRequest before use.
```

Reply:
This language feels a little too casual, I made some suggestions but I'm not sure if I'm 100% right mentioning `additionalInfo` please double check for accuracy.